paper.tex

```diff
@@ -298,6 +298,13 @@ \subsection{Development Services}
 Founded in January 2017, the Research Computing department of Princeton University has experienced a tremendous growth from the initial two FTEs to a total of 18 FTEs in the span of five years~\autocite{Cosden2022}.
 This growth is based on a continuous influx of new funded projects once successful projects showcase the additional value of RSE services to researchers.
 
+[Success Story]
+The University of Manchester Software and Data Science group has successfully established specialized development services within their institution:
+The "Mobile Development Service" \autocite{manchester_mobile} team consists of RSEs that focus solely on developing and deploying mobile apps.
+Without a central RSE department to anchor such specialized expertise, it would probably be infeasible to establish such a service.
+Also, having this expertise centralized allows for synergies in the deployment procudure for mobile apps:
+The RSE department can create institutional accounts with the app stores and manage the time consuming deployment process including hard-to-setup procedures like code signing.
+
```

Comment:
since these apps are now signed by the bigger university it also means, that the institution profits from this visibility, not only the developer, right?

Reply:
Added a sentence about this.